File: src/Blizka.App/Domain/Entities/Photo.cs
namespace Blizka.App.Domain.Entities;

public sealed class Photo
{
    public Guid Id { get; set; }

    public Guid UserId { get; set; }

    public User? User { get; set; }

    public string Url { get; set; } = string.Empty;

    public string ThumbnailUrl { get; set; } = string.Empty;

    public string MediumUrl { get; set; } = string.Empty;

    public int SortOrder { get; set; }

    public bool IsMain { get; set; }

    public DateTimeOffset CreatedAt { get; set; }
}

File: frontend/src/domains/photos/types/photo.ts
/** Сверено с backend: `Blizka.Api/Photos/PhotoDtos.cs`. */
export type Photo = {
  id: string
  url: string
  thumbnailUrl: string
  mediumUrl: string
  sortOrder: number
  isMain: boolean
  createdAt: string
}

/** Лимит из backend: `UploadPhoto` отвечает 422 на седьмое фото. */
export const MAX_PHOTOS = 6
